addons/project/models/res_config_settings.py

```diff
@@ -11,6 +11,7 @@ class ResConfigSettings(models.TransientModel):
     module_hr_timesheet = fields.Boolean(string="Task Logs")
     group_subtask_project = fields.Boolean("Sub-tasks", implied_group="project.group_subtask_project")
     group_project_rating = fields.Boolean("Customer Ratings", implied_group='project.group_project_rating')
+    group_project_stages = fields.Boolean("Project Stages", implied_group="project.group_project_stages")
```

Comment:
PO has been asked 😄

Reply:
The PO prefers that we keep the information even if the option is disabled (in case one would disable the feature as an accident), so no changes required 😄